ego: Signal Handling misses include

The compilation of the host code, especially the `ubertooth-ego.c` file, fails since 063b614b91fec412837e6919e2d8bfcb49206fe9. This can easyly be fixed by including `signa.h` in this file.

### Steps to reproduce
1. Check out repository's `master` branch, at 063b614b91fec412837e6919e2d8bfcb49206fe9 or later
2. Try to build the `host` code

### Expected behaviour
Compile the project successfully.

### Actual behaviour
Compilation failure, check the following _Output_ section.

### Output
```
/build/host/ubertooth-tools/src/ubertooth-ego.c:130:2: warning: implicit declaration of function 'signal'; did you mean 'strsignal'? [-Wimplicit-function-declaration]
  130 |  signal(SIGINT, quit);
      |  ^~~~~~
      |  strsignal
/build/host/ubertooth-tools/src/ubertooth-ego.c:130:9: error: 'SIGINT' undeclared (first use in this function)
  130 |  signal(SIGINT, quit);
      |         ^~~~~~
/build/host/ubertooth-tools/src/ubertooth-ego.c:130:9: note: each undeclared identifier is reported only once for each function it appears in
/build/host/ubertooth-tools/src/ubertooth-ego.c:131:9: error: 'SIGQUIT' undeclared (first use in this function)
  131 |  signal(SIGQUIT, quit);
      |         ^~~~~~~
/build/host/ubertooth-tools/src/ubertooth-ego.c:132:9: error: 'SIGTERM' undeclared (first use in this function)
  132 |  signal(SIGTERM, quit);
      |         ^~~~~~~
make[2]: *** [ubertooth-tools/src/CMakeFiles/ubertooth-ego.dir/build.make:63: ubertooth-tools/src/CMakeFiles/ubertooth-ego.dir/ubertooth-ego.c.o] Error 1
make[1]: *** [CMakeFiles/Makefile2:433: ubertooth-tools/src/CMakeFiles/ubertooth-ego.dir/all] Error 2
make[1]: *** Waiting for unfinished jobs....
```

--- a/host/ubertooth-tools/src/ubertooth-ego.c
+++ b/host/ubertooth-tools/src/ubertooth-ego.c
@@ -27,6 +27,7 @@
 #include <string.h>
 #include <unistd.h>
 #include <stdlib.h>
+#include <signal.h>
 
 ubertooth_t* ut = NULL;
 int running = 1;

